code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm

```diff
@@ -190,7 +190,7 @@ Difficulty: Medium
 		drakewalls += new /obj/effect/temp_visual/drakewall(T) // no people with lava immunity can just run away from the attack for free
 	var/list/indestructible_turfs = list()
 	for(var/turf/T in RANGE_TURFS(2, center))
-		if(istype(T, /turf/open/floor/indestructible))
+		if(istype(T, /turf/open/ind_floor))
```

Comment:
This is a change. I need to look into this.

Reply:
`/turf/simulated/floor/indestructible` in Paracode essentially has the same properties as `turf/unsimulated`. I moved the type to `turf/open/ind_floor`